### PR TITLE
locust: add page

### PR DESCRIPTION
**Pull request by pit1s**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/locust.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# locust
+
+> Load-testing tool to determine number of concurrent users a system can handle.
+
+- Load-test example.com with web interface using locustfile.py:
```
Comment (agnivade) [inline pages/common/locust.md]: Put example.com in quotes.

Comment (pit1s) [reply pages/common/locust.md]: Done.

File: pages/common/locust.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`locust --master --host={{http://example.com}}`
+
+- Connect locust slave to master:
+
+`locust --slave --host={{http://example.com}}`
+
+- Connect locust slave to master on a different machine:
+
+`locust --slave --master-host={{master_ip}} --host={{http://example.com}}`
```
Comment (agnivade) [inline pages/common/locust.md]: Should we change it to "master_hostname" because the option mentions "master-host" ?

Comment (pit1s) [reply pages/common/locust.md]: I've only used IP for my tests but you can do both. I'll change it for simplicity. Thanks for the feedback.

Comment (pit1s) [reply pages/common/locust.md]: Done

**Comment by sbrl** [comment]
Looks ok to me! Thanks @pit1s :smiley_cat:

**Comment by pit1s** [comment]
@sbrl Happy to help :+1:

**Pull request merged**